farcaster/hubs/values.yaml

```diff
@@ -1,17 +1,14 @@
-# based on https://github.com/bjw-s/helm-charts/
-
-# NOTE: THIS IS HEAVY WORK IN PROGESS TODO: TEST, FINISH, DOCUMENT
-
-# Hubble
 controllers:
-  hubble: 
+  main:
```

Comment:
@SamuelLHuber its a little picky about this, you can technically rename but i think it's easier not to, always use controller=main and container=main